### PR TITLE
Margarita_Scorodnikova

### PR DESCRIPTION
**Pull request by Margareeta**

### CODE REVIEW

**Comment by NikolaevArtem** [comment]
You have conflicts, please merge with Master

File: src/main/java/Homework1/Main.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+import java.util.Scanner;
+
+public class Main {
+
+
+    public static void main(String[] args) {
+
+
+        Scanner scanner = new Scanner(System.in);
```
Comment (NikolaevArtem) [inline src/main/java/Homework1/Main.java:11]: Please use args from the start of the application. No need to complicate it for now

Comment (Margareeta) [reply src/main/java/Homework1/Main.java:11]: It sais it can't automatically merge and that I have to create a pull request which I seem to have done.

Comment (Margareeta) [reply src/main/java/Homework1/Main.java:11]: And I also can't delete the old Readme file form the src directory. I think I need your approval.

**Pull request closed without merging**